### DISCUSSION TITLE
Bring rust-jsonrpc into the rust-bitcoin organisation

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#1116] [General]
Do you still want to bring the `rust-jsonrpc` crate over to the `rust-bitcoin` organisation @apoelstra? If so, is there anything that needs doing before we can do so?

**Comment by apoelstra** [comment]
Yes -- and no, I don't think there's anything else. Does anyone have objections to this?

**Reply by Kixunil** [reply]
Feels a bit strange having a crate not directly related to Bitcoin here but not really an objection.

**Reply by apoelstra** [reply]
Its sole purpose is to talk to Bitcoin Core with less than 100 dependencies.
